types/canonical_json.go

```diff
@@ -3,14 +3,13 @@ package types
 import (
 	"time"
 
-	"github.com/tendermint/go-amino"
 	cmn "github.com/tendermint/tmlibs/common"
 )
 
 // Canonical json is amino's json for structs with fields in alphabetical order
 
 // TimeFormat is used for generating the sigs
-const TimeFormat = amino.RFC3339Millis
+const TimeFormat = "2006-01-02T15:04:05.000Z"
```

Comment:
old version `const TimeFormat = amino.RFC3339Millis` is correct

Reply:
The version of amino this uses doesn't contain the constant anymore:
https://github.com/tendermint/go-amino/commit/f81fa38144591e3e0581724fc30e5db820aba1a5

Reply:
oh, sorry. this one --->>>
https://github.com/tendermint/tendermint/pull/1756/files#diff-2a5c02592829cd2adae17d586208b772

Reply:
Tests fail, if I do this change.